application/forms/Config/User/UserForm.php

```diff
@@ -17,6 +20,21 @@ class UserForm extends RepositoryForm
      */
     protected function createInsertElements(array $formData)
     {
+        $passwordPolicyObject = null;
+        $passwordPolicy = Config::app()->get(
+            'global',
+            'password_policy',
+            DefaultPasswordPolicy::class
+        );
+        if (isset($passwordPolicy)) {
```

Comment:
Now, after https://github.com/Icinga/icingaweb2/pull/5419#discussion_r2307135923, $passwordPolicy is always set.